languages/ruby/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -0,0 +1,61 @@
+## True, False, and Nil
+
+True and false logical states are represented with `true` and `false` in Ruby. These may either be used or returned as literals on their own, or as a result of logical or comparison methods.
+
+```ruby
+true_variable = true
+false_variable = false
+
+true && false
+# => false is returned as a result of the logical method
+
+1 < 2
+# => true is returned as a result of the comparison method
+```
+
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Ruby, `nil` is a value used to express the _absence_ of an object.
+
+```ruby
+# I do not have a favorite color
+favorite_color = nil
+```
+
+In other programming languages, `null` or `none` values might play a similar role.
+
+## _Truthy_ and _falsey_
+
+In Ruby, all objects evaluate to a _truthy_ or _falsey_ value when they are encountered in a boolean context (like an `if` expression). All data is considered _truthy_ **except** for `false` and `nil`.
+
+## Control flow
+
+_Truthy_ and _falsey_ values are useful in the context of control flow. Like in procedural languages, Ruby has an _if...else_ construct, but it is often more common to use _if_ as a modifier to control the evaluation of an expression.
```

Comment:
```suggestion
_Truthy_ and _falsey_ evaluations are useful in the context of control flow. Like in procedural languages, Ruby has an `if`...`else` construct.  It is common to use `if` as "guarding" statement.
```

Still needs some work as well,  I think.  (Writing is hard.)

Reply:
```suggestion
_Truthy_ and _falsey_ evaluations are useful in the context of control flow. Like in procedural languages, Ruby has an `if`...`else` construct, but it is often more common to use `if` as a "guarding" statement to modify the evaluation of an expression.
```

I wanted to have the word modify in here because that is [how they are referred to in the docs](https://docs.ruby-lang.org/en/2.0.0/syntax/control_expressions_rdoc.html#label-Modifier+if+and+unless)

Reply:
We can always patch the docs. ;)

`but it is often more common` to `it may be more common` (I have not measured, and so don't know, and it may be in one field and not another.)

Reply:
```suggestion
_Truthy_ and _falsey_ evaluations are useful in the context of control flow. Like in procedural languages, Ruby has an `if`...`else` construct, but it may be more common to use `if` as a "guarding" statement to modify the evaluation of an expression.
```